source/photo.html

```diff
@@ -127,16 +133,22 @@ <h2 class="visually hidden">Редактор фотографий PINK</h2>
 </main>
 <footer class="footer">
```

Comment:
По мимо глобального блока footer, рекомендую использовать микро блоки. В данном случае у тебя будет примерно такая структура 
```suggestion 
<footer class="footer">
  <div class="footer__logo footer-logo">
    <img class="footer-logo__img" src="img/logo-pink-blue-mobile.svg" width="147" height="40" alt="логотип PINK">
  </div>
  <ul class="footer__social footer-social">
    <li class="footer-social__item twitter-link">
      <a href="#" class="footer-social__link">
        <img class="footer-social__img footer-social__img--twitter" src="img/icon-twitter.svg" width="19" height="15" alt="логотип twitter">
      </a>
    </li>
  </ul>
  <div class="footer__academy footer-academy">
    <p class="footer-academy__text">Разработано</p>
    <img class="footer-academy__img" src="https://placeimg.com/27/34/any" alt="логотип academyacademy">
    <img class="footer-academy__img" src="img/logo-htmlacademy.svg" width="27" height="34" alt="логотип htmlacademy">
  </div>
</footer>
```